.stylelintrc.js

```diff
@@ -1,4 +1,38 @@
+// .stylelintrc.js
+function autoFixFunc(node, validation, root, config) {
+  const { value, prop } = node
+
+  if (prop === 'color') {
+    switch (value) {
+      case '#fff':
+        // auto-fix by returned value
+        return '$color-white'
+
+      case 'red':
+        // auto-fix by PostCSS AST tranformation
+        node.value = '$color-red'
+
+      default:
+        // optional, you can throw your own error message if the value is not stated or handled, ex: color: blue
+        throw `Property ${prop} with value ${value} can't be autofixed!`
+        // or an Error object
+        throw new Error(`Property ${prop} with value ${value} can't be autofixed!`)
+        // or a falsy value to use the default error message
+        throw null;
+    }
+  }
+}
+
 module.exports = {
-  // See https://github.com/wagtail/stylelint-config-wagtail for rules.
-  extends: '@wagtail/stylelint-config-wagtail',
-};
+// "See https://github.com/wagtail/stylelint-config-wagtail for rules." //,
```

Comment:
@MebinAbraham and @AWitcherONS please note that I am working on some overrides to the wagtail stylelint config as part of my front-end set up code. It would be preferable to avoid changing any scss and js linting as part of the megalinter changes, and let stylelint and eslint handle it if possible.

Reply:
Should no longer be a problem; CSS/Js is handled natively and not via Megalinter.